main.rb

```diff
@@ -0,0 +1,64 @@
+require 'json'
+require_relative 'App/Game/game'
+require_relative 'App/Author/author'
+require_relative 'App/Item/item'
+require_relative 'App/MusicAlbum/music_album'
+require_relative 'App/Book/book_label'
+
+MusicAlbum.load_albums_from_json
+
+book_label = BookLabel.new
+# ALBUMS_FILE = 'albums.json'.freeze
+# MOVIES_FILE = 'movies.json'.freeze
+# GAMES_FILE = 'games.json'.freeze
+# AUTHORS_FILE = 'authors.json'.freeze
+
+loop do
+  puts "\nOptions:"
+  puts '1. List all books'
+  puts '2. List all music albums'
+  puts '3. List all movies'
+  puts '4. List all games'
+  puts '5. List all genres'
+  puts '6. List all labels'
+  puts '7. List all authors'
+  puts '8. Add a book'
+  puts '9. Add a music album'
+  puts '10. Add a movie'

```

Comment:
- [ ] @Team great work on your UI however I would recommend removing options 3 and 10 since their functionality is not implemented the are only suppose to be there for a team of 4 people.

Reply:
finished! ✔️